.github/workflows/github_push.yaml

```diff
@@ -1,68 +1,79 @@
-name: push
-on: [push]
+# Github worflow to test Agora on a variety of platforms
+#
+# The CI triggers on 'push' to upstream.
+#
+# Note:
+# - Try to use the native Github action syntax (${{ expression }}) when possible,
+#   as they are substituted with their value in the logs, unlike env variable.
+#   For example use `${{ github.workspace }}` over `${GITHUB_WORKSPACE}`
+name: CD pipeline
```

Comment:
Could add a comment that the D in CD refers to the documentation being auto-deployed.

Reply:
It refers to "deployment" as in "Continuous deployment". Couldn't find a cool name. Suggestions welcome.

Reply:
oh I know, but mentioning what is actually deployed would be nice.